maven-archetype-plugin/src/main/java/org/apache/maven/archetype/ui/ArchetypeConfiguration.java

```diff
@@ -19,12 +19,15 @@
  * under the License.
  */
 
+import org.apache.maven.archetype.common.Constants;
+import org.codehaus.plexus.util.StringUtils;
```

Comment:
This is still an unrelated change because those imports are already present a few lines below and have been moved to this position. An unrelated change for me.